### PR TITLE
Task: Design inspection

### PR DESCRIPTION
**Pull request by paulathevalley**
Resolves https://github.com/FormidableLabs/victory-docs/issues/148

(WIP)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can do this on your own if you'd like, or you can...
 * ```cd victory-tutorial```
 * Replace the existing code in the ```client.js``` file with:
 
-```
+```js
```
Comment (bmathews) [inline docs/index.md]: is this fine as js, or should it be jsx?

Comment (paulathevalley) [inline docs/index.md]: Yes. I tried `jsx`, and there wasn't any syntax highlighting for some unknown reason, but `js` does have syntax highlighting.

File: src/screens/home/components/benefits.js [hunk 2]
```diff
@@ -89,83 +43,85 @@ class Benefits extends React.Component {
...
-            marginRight: "3vw",
-            marginLeft: "3vw",
-            textAlign: "center"
-          }}
-        >
-          <RadiumLink style={styles.link} to="/docs">
-            Get Started <Icon glyph="internal-link" />
-          </RadiumLink>
+          <div className="Benefits-btn">
+            <RadiumLink style={styles.link} to="/docs">
```
Comment (bmathews) [inline src/screens/home/components/benefits.js]: Seems strange that this a still RadiumLink, but the rest have been pulled out?

Comment (paulathevalley) [inline src/screens/home/components/benefits.js]: Agreed! This particular link has a background image with an `svg`, which requires an update to the archetype with the post-css svg support you mentioned, which I haven't gotten around to doing. So, I left it in a 'working state' for now... 

There is still some Radium tucked away in certain components. Ideally, it'll all be ripped out, but the tight deadline meant I couldn't get to everything at once. :(

Comment (bmathews) [inline src/screens/home/components/benefits.js]: :100: :shipit: :shipit: 🚢  🚀

**Pull request merged**